all-actors/src/main/java/org/sunbird/actor/CertBackgroundActor.java

```diff
@@ -31,11 +31,22 @@ public void onReceive(Request request) throws Throwable {
                 add(request);
                 break;
 
+            case "delete_cert_es":
```

Comment:
Delete is not a separate call, During add certificate if user pass oldId then for that user provided certificate will be deleted.

Reply:
for cassandra delete happen along with add , but ES delete will run in background